installer/templates/phx_assets/tailwind_heroicons.js

```diff
@@ -0,0 +1,43 @@
+const plugin = require("tailwindcss/plugin")
```

Comment:
I wonder if this should be inside the css directory?

In any case:

```suggestion
// A Tailwind plugin that makes "hero-#{ICON}" classes available
const plugin = require("tailwindcss/plugin")
```